### ISSUE TITLE
`ArrayIndexOutOfBoundsException: Index 36 out of bounds for length 36` while call `CellBuilder.bits.toBooleanArray()`

### ISSUE THREAD

**Issue by andreypfau** [andreypfau/ton-kotlin#19]
CellBuilder has size 1023 and bits too, but for some reason BitString has only 36 bytearray, instead 128

```
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 36 out of bounds for length 36
	at org.ton.bitstring.ByteBackedBitString$Companion.get(ByteBackedBitString.kt:174)
	at org.ton.bitstring.ByteBackedBitString.get(ByteBackedBitString.kt)
	at org.ton.bitstring.ByteBackedBitString.getOrNull(ByteBackedBitString.kt:15)
	at org.ton.bitstring.ByteBackedBitString.get(ByteBackedBitString.kt:12)
	at org.ton.bitstring.ByteBackedBitString$BitStringIterator.next(ByteBackedBitString.kt:105)
	at org.ton.bitstring.ByteBackedBitString$BitStringIterator.next(ByteBackedBitString.kt:99)
	at kotlin.collections.CollectionsKt___CollectionsKt.toBooleanArray(_Collections.kt:1086)
	at org.ton.bitstring.ByteBackedBitString.toBooleanArray(ByteBackedBitString.kt:31)
```

**Comment by fliesonacid** [comment]
Tried to reproduce the issue with test like this
  ```
    fun `issue 19 test`() {
        val cellb = CellBuilder.beginCell()
            .storeUInt(1000, 256)
            .storeUInt(2555, 256)
            .storeUInt(48675, 256)
            .storeUInt(61234, 255)
        cellb.bits.toBooleanArray().forEach { println(it) }
    }
```
No ArrayIndexOutOfBoundsException was thrown. Am I missing something?

**Comment by andreypfau** [comment]
Fixed in [cd9d50c](https://github.com/andreypfau/ton-kotlin/commit/cd9d50c9dd77a0d92c5c67295ec54f01108a98b0)
